launch.go

```diff
@@ -494,7 +494,9 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	}
 
 	// Initialize pr comments (Issue #1858)
-	delete(mergedMeta, "summary")
+	if _, ok := mergedMeta["meta"]; ok {
+		delete(mergedMeta["meta"], "summary")
```

Comment:
Probably need to cast it like this
```suggestion
	if metadata, ok := mergedMeta["meta"]; ok {
		delete(metadata.(map[string]interface{}), "summary")
```
https://github.com/screwdriver-cd/launcher/pull/383/files#diff-0952bc032b3245f7e7282a5d24e8a1775eb3e1f265de43bb26834860e6acc4edR318

Reply:
Thanks!